app/src/main/java/com/sunlightlabs/android/congress/notifications/subscribers/VotesBillSubscriber.java

```diff
@@ -16,7 +16,7 @@
 
 	@Override
 	public String decodeId(Object result) {
-		return String.valueOf(((Bill.Vote) result).voted_at.getTime());
+		return String.valueOf(((Bill.Vote) result).voted_on.getTime());
```

Comment:
Similar problem as actions, but in this case we do have a `time` field available that we're not taking advantage of. So we should turn `voted_on` back into `voted_at` and parse that field properly.